.travis.yml

```diff
@@ -10,7 +10,10 @@ addons:
   postgresql: "9.3"
 before_script:
   - gem install pg
-  - psql -c 'create database travis_ci_test;' -U postgres
+  - psql -c "CREATE USER ox_exchange WITH CREATEDB PASSWORD 'ox_exchange'" -U postgres
+  - psql -c 'CREATE DATABASE ox_exchange_test' -U postgres
+  - psql -c 'GRANT ALL PRIVILEGES ON DATABASE ox_exchange_test TO ox_exchange' -U postgres
+  - RAILS_ENV=test bundle exec rake db:migrate
 script:
   - RAILS_ENV=test bundle exec rake --trace
 branches:
```

Comment:
Yeah that's a great idea.  That'll validate that the schema.rb file is correct.  Of course it always **should** be but that never gets tested otherwise.